cli/container.go

```diff
@@ -12,17 +10,6 @@ type container struct {
 	volume []string
 }
 
-func (c *container) init() *cobra.Command {
-	cmd := &cobra.Command{}
-
-	// TODO add flag
-	cmd.Flags().StringVar(&c.name, "name", "", "specified the container's name")
-	cmd.Flags().BoolVarP(&c.tty, "tty", "t", false, "allocate a tty device")
-	cmd.Flags().StringSliceVarP(&c.volume, "volume", "v", nil, "create container with volumes")
-
-	return cmd
-}
-
```

Comment:
Define the container structure and flags, the reason is that these flags will be used to implement "create" and "run" commands. If move the flags into "create" command, the "run" command will not reuse it. You should rewrite all flags.

Reply:
Thanks a lot for your feedback. Actually I totally agree with what you said. While yesterday when I try to find container name flag to debug, I spend a few seconds to make it. So maybe it is not so straight forward. Leaving them in container command makes it much more clear. In the future when we support run. I suggest we can add them in create and run's addFlags. And then it will make developers clear and this will not violate "DRY" rule. 

WDYT? @skoo87

Reply:
Now is ok, In the future, I suggest we use a common function that can be called by "create" and "run" to add flags.  @allencloud